test/Makefile.in

```diff
@@ -123,7 +123,8 @@ testautomation$(EXE): $(srcdir)/testautomation.c \
 		      $(srcdir)/testautomation_syswm.c \
 		      $(srcdir)/testautomation_timer.c \
 		      $(srcdir)/testautomation_video.c \
-		      $(srcdir)/testautomation_hints.c
+		      $(srcdir)/testautomation_hints.c \
+					$(srcdir)/testautomation_math.c
```

Comment:
This should use spaces instead of tabs, for consistency.

Reply:
My bad, I didn't see the difference in my text editor, this is fixed in the latest commit.